app.js

```diff
@@ -100,7 +100,7 @@ fsTools.findSorted('public/data', /[^.]+\.metadata.json/, function(err, files) {
     var locationName = path.basename(metadataFilePath, '.metadata.json');
 
     // Exclude template file
-    if (metadataFilePath.indexOf('/_') == -1) {
+    if (metadataFilePath.indexOf('/') == -1) {
```

Comment:
We need this to exclude the paths `public/data/_TEMPLATE.metadata.json` and `public/data/_work-in-progress/...`.